services/users/src/models/profile.ts

```diff
@@ -9,6 +11,7 @@ export interface Profile {
   };
   phoneNumber: string;
   gender: string;
+  resume: File;
```

Comment:
Here, you can just use the `Types.ObjectId` as we use in other places in the repo, as this field would just refer to an object. Then, when you want to get a user's resume, you would make a call with this id.